spec/order_spec.rb

```diff
@@ -0,0 +1,36 @@
+require './lib/order'
+require './lib/restaurant'
+
+describe "Order" do
+
+	it 'should start with an empty hash' do
+		order = Order.new
+		expect(order.my_order).to be_empty
+	end
+
+	it 'should have an item in the order hash' do
+		order = Order.new
+		pizza = Restaurant.new
+		pizza.add("margarita", 10)
+		pizza.add("pepperoni", 12)
+		pizza.add("hawaiian", 12)
+		order.select(pizza,"margarita")
+		expect(order.my_order).not_to be_empty
+	end
```

Comment:
I thought a 'double' may be a good option here because you would be calling from another class the "Dish", and then possibly not have to repeat.

Reply:
I've been having trouble understanding how to use doubles tbh. But after viewing your code, I think I understand them bit better.